bin/osg-notify

```diff
@@ -185,10 +198,23 @@ def main():
                     emails.add(contact['Email'])
         recipients.update(emails)
     if args.oim_recipients and 'resources' in args.oim_recipients:
-        if args.fqdn_filter:
-            results = topology_utils.get_resource_contacts_by_fqdn(args)
-        else:
-            results = topology_utils.get_resource_contacts(args)
+        attempts = 3
+        while attempts > 0:
+            try:
+                if args.fqdn_filter:
+                    results = topology_utils.get_resource_contacts_by_fqdn(args)
+                else:
+                    results = topology_utils.get_resource_contacts(args)
+            except topology_utils.InvalidPathError as exc:
+                print(exc)
+                exit(1)
```

Comment:
```suggestion
                exit(str(exc))
```

This can be simplified. For example:

```
blin@blin-latitude:~$ python
Python 3.9.7 (default, Aug 30 2021, 00:00:00) 
[GCC 10.3.1 20210422 (Red Hat 10.3.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> exit('asdfasdf')
asdfasdf
blin@blin-latitude:~$ echo $?
1
```